symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -201,29 +203,69 @@ public V4Message send(@Nonnull V4Stream stream, @Nonnull Message message) {
   @Override
   public V4Message send(@Nonnull String streamId, @Nonnull Message message) {
     return this.executeAndRetry("send", () ->
-        this.doSend(streamId, message, this.authSession.getSessionToken(), this.authSession.getKeyManagerToken())
+        this.doSendMessage(streamId, message)
     );
   }
 
+  /**
+   * Sends a message to multiple existing streams.
+   *
+   * @param streamIds the list of stream IDs to send the message to
+   * @param message   the message to be sent
+   * @return a {@link V4MessageBlastResponse} object containing the details of the sent messages
+   * @see <a href="https://developers.symphony.com/restapi/reference#create-message-v4">Create Message v4</a>
+   */
+  public V4MessageBlastResponse blastMessage(@Nonnull List<String> streamIds, @Nonnull Message message) {
```

Comment:
I'm wondering about the method name vs the existing methods, we could reuse send() but the returned response is a bit weird and maybe we want to make it very explicit it is a blast message
maybe sendBlast? (just to be more consistent with the others send)